### PR TITLE
Feature/hw 8

### PR DESCRIPTION
**Pull request by LevapVeeskela**
imitation to registration in airport Minsk-2

### CODE REVIEW

File: ItAcademyHomeWorks/Infrastructure/Abstractions/Services/HW_08/ICheckInService.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+namespace Infrastructure.Abstractions.Services.HW_08
+{
+    public interface ICheckInService
+    {
+        bool CheckTickets(PersonModel person);
+
+        void SetTickets(PersonModel person);
+
+        bool ClaimBaggage(PersonModel person);
+
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/Infrastructure/Abstractions/Services/HW_08/ICheckInService.cs]: just one empty line could be deleted

File: ItAcademyHomeWorks/Common/Models/PersonModel.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+{
+    public class PersonModel
+    {
+        public string Passport { get; set; }
+
+        public bool IsHaveTickets { get; set; }
+
+        public bool IsHaveBaggage { get; set; }
+
+        public ushort? WeightBags { get; set; }
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/Common/Models/PersonModel.cs]: I see Nullable type there. good point.
As for the naming - I would rename it. This name is not clear for me

File: ItAcademyHomeWorks/Common/Constants/Constants.cs [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            public const string TextStageSPassportControlInfo = "Этап проверки личности и визы";
+            public const string TextImitationCheckPassportInBdInfo = "Иммитируется проверка в БД паспартов.";
+            public const string TextConfirmationVisaInfo = "Нужна ли виза в {0}";
+        }
+        
+        public static class RegistrationBase
+        {
+            public const string PatternTextSuccessFlyInfo = "{0} успешно прошёл все тягости регистрации и улетел в {1}, попивать вискарёк в бассейне!";
+            public const string PatternTextRejectRegistrationInfo = "{0} не прошёл регистрацию и не улетел в {1}";
+        }
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/Common/Constants/Constants.cs]: these all constants I would move to one parent static class, for instance **AirportRegistration**

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_08/RegistrationBaseService.cs [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+using Infrastructure.Abstractions.Services.HW_08;
+
+namespace BusinessLogic.Services.HW_08
+{
+    public class RegistrationBaseService
+    {
+        private readonly IConfigurationPersonService _cps;
+        private readonly ICheckInService _cis;
+        private readonly ISecurityCheckService _scs;
+        private readonly IPassportControlService _pcs;
```
Comment (VadzimPapko) [inline ItAcademyHomeWorks/BusinessLogic/Services/HW_08/RegistrationBaseService.cs]: your private fields names are look the same. so, it is hard to understand what those do really mean.
Let's imagine you have a bunch of methods where these fields will be used. I am sure 100% in a time (one year) you will go to declaration of this fields just to know what it is. 

I would rename those a bit

File: ItAcademyHomeWorks/Common/Models/PersonModel.cs (hunk 2 above)
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/Common/Models/PersonModel.cs]: Вес сумок, вроде же понятно всё) сумок может быть и не быть, поэтому nullable type)

File: ItAcademyHomeWorks/BusinessLogic/Services/HW_08/RegistrationBaseService.cs (hunk 4 above)
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/BusinessLogic/Services/HW_08/RegistrationBaseService.cs]: this is hw only example, so I didn't bother very much about names... but I will consider in the future, thanks)

File: ItAcademyHomeWorks/Common/Constants/Constants.cs (hunk 3 above)
Comment (LevapVeeskela) [reply ItAcademyHomeWorks/Common/Constants/Constants.cs]: ok, in next hw

**Pull request merged**